index.html

```diff
@@ -29,22 +30,205 @@ <h1>
+          </ul>
+          <div class="button"><a href="#">See project</a></div>
+        </div>
+        </div>
+<!--2-->
+        <div class="item">
+          <div class="border">
+            <div class="border-image">
+            <img src="images/grid.svg" alt="">
+          </div>
+          </div>
+          <div class="content">
+            <div class="title"><p>
+          Multi-Post Stories <br>Gain+Glory
+          </p></div>
+          <ul class="links">
+            <li><a href="#">Ruby on Rails</a></li>
+            <li><a href="#">CSS</a></li>
+            <li><a href="#">JavaScript</a></li>
+            <li><a href="#">HTML</a></li>
+        </ul>
+        <div class="button"><a href="#">See project</a></div>
+      </div>
+      </div>
+<!--3-->
+      <div class="item">
+        <div class="border">
+          <div class="border-image">
+          <img src="images/grid.svg" alt="">
+        </div>
+        </div>
+        <div class="content">
+          <div class="title"><p>
+            Multi-Post Stories <br>Gain+Glory
+            </p></div> 
+        <ul class="links">
+          <li><a href="#">Ruby on Rails</a></li>
+          <li><a href="#">CSS</a></li>
+          <li><a href="#">JavaScript</a></li>
+          <li><a href="#">HTML</a></li>
+      </ul>   
+      <div class="button"><a href="#">See project</a></div>
+    </div>
+
+<!--4-->
+    <div class="item">
+      <div class="border">
+        <div class="border-image">
+        <img src="images/grid.svg" alt="">
+      </div>
+      </div>
+      <div class="content">
+        <div class="title"><p>
+          Multi-Post Stories <br>Gain+Glory
+          </p></div>
+      <ul class="links">
+        <li><a href="#">Ruby on Rails</a></li>
+        <li><a href="#">CSS</a></li>
+        <li><a href="#">JavaScript</a></li>
+        <li><a href="#">HTML</a></li>
+    </ul>
+    <div class="button"><a href="#">See project</a></div>
+  </div>
+
+<!--5-->
+  <div class="item">
+    <div class="border">
+      <div class="border-image">
+      <img src="images/grid.svg" alt="">
+    </div>
+    </div>
+    <div class="content">
+      <div class="title"><p>
+        Multi-Post Stories <br>Gain+Glory
+        </p></div>
+    <ul class="links">
+      <li><a href="#">Ruby on Rails</a></li>
+      <li><a href="#">CSS</a></li>
+      <li><a href="#">JavaScript</a></li>
+      <li><a href="#">HTML</a></li>
+  </ul>
+  <div class="button"><a href="#">See project</a></div>
+</div>
+
+<!--6-->
+<div class="item">
+  <div class="border">
+    <div class="border-image">
+    <img src="images/grid.svg" alt="">
+  </div>
+  </div>
+  <div class="content">
+    <div class="title"><p>
+      Multi-Post Stories <br>Gain+Glory
+      </p></div>
+  <ul class="links">
+    <li><a href="#">Ruby on Rails</a></li>
+    <li><a href="#">CSS</a></li>
+    <li><a href="#">JavaScript</a></li>
+    <li><a href="#">HTML</a></li>
+</ul>
+<div class="button"><a href="#">See project</a></div>
```

Comment:
- [ ] You did a great job using the `a tag` here for the see project button. 👍 But there are some changes you need to make here, The hover state of your button is different from what is provided in the Figma design, Please you need to fix this as your project is meant to look identical to the Figma design. 😄 
- Your hover state:
![Screenshot 2023-04-01 215726](https://user-images.githubusercontent.com/107632780/229314236-732af8bb-d833-4977-a943-b366952ff17b.png)
- Figma design button states:
![Screenshot 2023-04-01 215544](https://user-images.githubusercontent.com/107632780/229314244-8eda4a24-45d1-4f18-96dd-62f123ea6861.png)